mappings/net/minecraft/text/component/KeyBindComponent.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_scqpkchq net/minecraft/text/component/KeybindComponent
+CLASS net/minecraft/unmapped/C_scqpkchq net/minecraft/text/component/KeyBindComponent
```

Comment:
if we're sticking with this refactor please remove `keybind` from the dictionary

Reply:
oh, will do, since yeah, with PR #70? the standard is key binds, not key bindings nor keybinds